pkg/virtctl/ssh/native.go

```diff
@@ -44,9 +43,9 @@ type NativeSSHConnection struct {
 	Options      SSHOptions
 }
 
-func (o *SSH) nativeSSH(kind, namespace, name string) error {
+func (o *SSH) nativeSSH(kind, namespace, name string, clientConfig clientcmd.ClientConfig) error {
```

Comment:
Same question as above if we can storage the command in the SSH struct to avoid the additional parameter to the function